svalbard/odata/src/main/java/org/n52/svalbard/odata/grammar/STAQueryOptionsGrammar.interp

```diff
@@ -0,0 +1,342 @@
+token literal names:
```

Comment:
These should be generated during the build and not added to source control.

Reply:
removed in d15ad2d